### PR TITLE
[HOPSWORKS-954] Add support for ROCm

### PR DESCRIPTION
**Pull request by robzor92**
## Make sure there is no duplicate PR for this issue

* **Please check if the PR meets the following requirements**
- [ ] Adds tests for the submitted changes (for bug fixes & features)
- [ ] Passes the tests
- [ ] HOPSWORKS JIRA issue has been opened for this PR
- [ ] All commits have been squashed down to a single commit

* **Post a link to the associated JIRA issue**

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

* **Other information**:

### CODE REVIEW

**Comment by robzor92** [comment]
https://github.com/logicalclocks/hops-testing/pull/360

**Comment by robzor92** [comment]
@tkakantousis @SirOibaf  please review.

File: hopsworks-common/src/main/java/io/hops/hopsworks/common/util/SparkConfigurationUtil.java [hunk 1]
```diff
@@ -147,6 +155,7 @@
                             experimentType != null)));
 
     if(experimentType != null) {
+      addToSparkEnvironment(sparkProps,"LC_ALL", "C", settings);
```
Comment (tkakantousis) [inline hopsworks-common/src/main/java/io/hops/hopsworks/common/util/SparkConfigurationUtil.java]: @robzor92 are these properties somewhere documented?

Comment (robzor92) [reply hopsworks-common/src/main/java/io/hops/hopsworks/common/util/SparkConfigurationUtil.java]: I ended up skipping setting LC_ALL. This setting is supposed to force all locale settings to be the same, for example en_US.UTF-8. I had to set LC_ALL=C to get rocm to work, now it seems I can skip setting it. However we have issues with locale on other installations (TF doesn't like swedish) so it needs to be managed at some point. For example we should maybe set all locale aspects to be en_US.UTF-8 in chef. Added link to docs for CUDA_VISIBLE_DEVICES and HIP_VISIBLE_DEVICES.

**Pull request merged**